pkg/daemon/update.go

```diff
@@ -357,22 +364,41 @@ func (dn *CoreOSDaemon) applyOSChanges(mcDiff machineConfigDiff, oldConfig, newC
 		if dn.nodeWriter != nil {
 			dn.nodeWriter.Eventf(corev1.EventTypeNormal, "InClusterUpgrade", fmt.Sprintf("Updating from oscontainer %s", newConfig.Spec.OSImageURL))
 		}
-		var err error
-		if osImageContentDir, err = ExtractOSImage(newConfig.Spec.OSImageURL); err != nil {
+
+		if isLayeredImage, err = client.IsBootableImage(newConfig.Spec.OSImageURL); err != nil {
+			// TODO(jkyros): Maybe this shouldn't be an error for now, so we don't impact any
```

Comment:
In theory this should only return err if the image itself can't be pulled right? Which would be a failure anyways?

Probably should wrap the error with some context though

Reply:
You're correct, I was just paranoid about adding additional error possibility for people who weren't using the feature. I added some context to the error.